### PR TITLE
Remove old stuff and the python .bin data loader.

### LINKED ISSUE #116: Remove the python data loader?
Maybe we should consider removing the python data loader? It's not usable in practical terms, and drags behind the need of updating it along the c++ data loader.

### PR DESCRIPTION
**Pull request by Sopel97**
Closes #116

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from feature_transformer import DoubleFeatureTransformerSlice
 
 # 3 layer fully connected network
-L1 = 512
+L1 = 64
```
Comment (vondele) [inline model.py]: probably not intended ?

Comment (Sopel97) [reply model.py]: obviously lol thanks for catching that

**Comment by vondele** [comment]
I think this can be merged, was waiting for @glinscott for feedback.

**Comment by glinscott** [comment]
Ah, sorry for the delay, yup, lgtm.

**Pull request merged**